.pipelines/e2e-job-azure.yaml

```diff
@@ -0,0 +1,45 @@
+parameters:
+  name: ''
+
+jobs:
+- job: ${{ parameters.name }}
+  dependsOn:
+  - unit_tests
+  - e2e_test_kind
+  timeoutInMinutes: 120
```

Comment:
We should probably timeout alot sooner than 120mins right?

Reply:
Sure, we can bring it down to 30m given cluster creation time is < 15 mins on average.